### ISSUE TITLE
Map doesn't render correctly for zoom meetings

### ISSUE THREAD

**Issue by jeznag** [code4recovery/tsml-ui#323]
I have to put in a rough address for zoom meetings otherwise the meeting doesn't appear in the list but then it shows a grey box where the map should be. I don't want to put an actual address in because then people might think it's a face to face meeting.

Example: https://debtorsanonymous.org.au/?meeting=carlisle-perth-da

Data source: https://docs.google.com/spreadsheets/d/1YRpUph8Xfw-hX-2IegFx_S4YQUprFTqLPYxfL42Pb98/edit#gid=0

![CleanShot 2023-08-06 at 11 43 46@2x](https://github.com/code4recovery/tsml-ui/assets/1034890/350d06d3-552c-441d-8971-781bb5c19798)

**Comment by joshreisner** [comment]
that's right. if there is no exact address the map does not show. some groups like to show a static image there, [for example](https://sussexaa.org/meetings/?meeting=spiritual-awakening)

i had thought to perhaps show a general area map when it sees four coordinates like your sheet has, but once i implemented it it was just confusing. it looked like there was an error and the pin was missing because of some technical glitch rather than intentionally

**Comment by jeznag** [comment]
That image would work. How do I configure that using the JS version?

**Comment by joshreisner** [comment]
ok great try adding this CSS code

https://github.com/code4recovery/12-step-meeting-list/discussions/922#discussioncomment-3646436

```
#tsml-ui .meeting.online:not(.in-person) .map::after {
  background-image: url(https://images.unsplash.com/photo-1588196749597-9ff075ee6b5b?crop=entropy&cs=tinysrgb&fit=crop&fm=jpg&h=1440&ixid=MnwxfDB8MXxhbGx8fHx8fHx8fHwxNjIyMTIzODkw&ixlib=rb-1.2.1&q=80&utm_campaign=api-credit&utm_medium=referral&utm_source=unsplash_source&w=1920);
  background-position: center center;
  background-repeat: no-repeat;
  background-size: cover;
  bottom: 0;
  content: '';
  left: 0;
  opacity: 0.25;
  position: absolute;
  right: 0;
  top: 0;
}
```

**Comment by jeznag** [comment]
Nice one thanks.

**Comment by clickonetwo** [comment]
@joshreisner The above CSS is not working for me.  When I inspect the meeting page I can see the div with id tsml-ui and it has classes meeting and online but there is no descendant with class map.  Has something changed in the design or am I missing something?

**Comment by clickonetwo** [comment]
Ah, but then again I'm not seeing the maps next to in-person meetings either.  So apparently my configuration is off.

**Comment by clickonetwo** [comment]
Yep, when I use a Mapbox API key rather than a Google API key everything works great.  Sorry for the noise.

**Comment by jeznag** [comment]
![Image](https://github.com/user-attachments/assets/e3f10720-8d83-4a3d-a7b0-cce425c90a01)

This doesn't seem to be adding the .meeting.online class anymore. For some reason this meeting shows up as in person when I don't want it to.

https://www.debtorsanonymousaustralia.org/meetings/find-a-meeting -> [Zoom] Creative BDA (BDA #110115}

I've tried adding "Location temporarily unavailable" type which kills the map but people complained that they don't like that.

The HTML doesn't have any classes that I can access. Is there any workaround other than adding a javascript mutation observer that looks for "in person" and removes it?

**Comment by joshreisner** [comment]
@jeznag looks like the coordinates for this meeting are `-33.85891,151.20876` - this is a point, which implies an in-person meeting to TSML UI.

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/fe26cff8-3184-4b6b-bfcd-984499a27fec" />

if you can use a boundary instead (has four decimals separated by three commas) then this will show up as an online meeting - you could potentially just copy the value from your `[ZOOM] Waverley Syd DA` meeting

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/17649b27-97ee-4e4c-98f7-be71792e7897" />
